### PR TITLE
Homepage having the title and image of it and blank page

### PR DESCRIPTION
**Pull request by GeethaKamala**

### CODE REVIEW

**Comment by LakshmiSowndarya** [comment]
Please add some description for every commit

File: templates/HomePage.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title> E-Booking </title>
```
Comment (LakshmiSowndarya) [inline templates/HomePage.html]: Change the title to Railway E-Booking

Comment (LakshmiSowndarya) [inline templates/HomePage.html]: Attach the UI, when you are working on it. So that the user can understand the user experience

Comment (GeethaKamala) [inline templates/HomePage.html]: UI of title:
image

Comment (LakshmiSowndarya) [inline templates/HomePage.html]: I am fine with this title but later on we can change the icon of the page

**Pull request merged**